pie/utils/objects.py

```diff
@@ -63,6 +57,14 @@ def __init__(
             )
         self.add_item(self.lock_button)
 
+        self.add_item(
+            nextcord.ui.Button(
+                label="\u25b7",
+                style=nextcord.ButtonStyle.green,
+                custom_id="right-button",
+            )
+        )
+
     def __repr__(self):
```

Comment:
> it looks overall better

Does it though? You are mixing two functionalities together -- scrolling and locking. I think having it behind these two had its reasons. I get that it's nice and symmetric now, but I disagree.

Reply:
Well, by the opinion of people from server that were talking about this it does. Having it behind those two has IMHO  no special reason.

Reply:
OK, so - they are not two completly different functionalities, they work together. So IMHO it's not that big deal to have it inbetween. Also, it looks kinda awful when you have under this other buttons that are in some way symetric, such as Voting buttons.

Reply:
![image](https://user-images.githubusercontent.com/6602975/152540217-85bb3a05-405a-4d20-bd9b-a91deea5e078.png)

Reply:
Alright.